OneCommunityGlobal/HighestGoodNetworkApp
# Jiayu --> Demi_create_no_records_state

Create a no records state.

# Description
Implements Phase 2 WBS 3.3.9: Adds no records state to records modal.

## Related PRS (if any):
none

## Main changes explained:
- Update records modal so that Update records and Purchase records both contain functionality for if no records are present
…

## How to test:
1. git pull
2. check into current branch
3. do npm install and npm run start:local to run this PR locally
4. Clear site data/cache in local storage
5. log in as any user
6. go to http://localhost:3000/bmdashboard/login to confirm your login credentials
7. go to http://localhost:3000/bmdashboard/materials
8. select "View" under "Purchases"
9. Verify that if no records are present modal reads "There are no purchase records"
10. select "View" under "Updates"
11. Verify that if no records are present modal reads "There are no updates to display"

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/93569791/fee3d265-9535-43bf-a575-60fc5dd54db5


## Note:
Use development branch for back end.
